### PR TITLE
Update Provider version to v1.0.16

### LINKED ISSUE #41: Update provider v4 docs
**Is your feature request related to a problem? Please describe.**
Update provider v4 docs

**Describe the solution you'd like**
NA

**Describe alternatives you've considered**
NA

**Additional context**
NA

### PR DESCRIPTION
**Pull request by akshay-ap**
Fixes #41

Related to https://github.com/oceanprotocol/docs/issues/1007

Changes proposed in this PR:

- Provider version to v1.0.16

### CODE REVIEW

**Review by MantisClone — COMMENTED** [review]
LGTM. @akshay-ap I think you'll have to wait for someone else's approval given that I'm not part of the core team anymore.

**Review by LoznianuAnamaria — APPROVED** [review]

**Pull request merged**
